tests/phpunit/class-wc-stripe-level3-data-test.php

```
@@ -297,6 +297,86 @@ public function test_level3_data_with_three_decimal_shipping() {
+		// The Level 3 total must equal the capture amount (order total 15.60 => 1560).
+		$level3_total  = array_reduce(
+			$result['line_items'],
+			function ( $sum, $item ) {
+				return $sum + ( $item->quantity * $item->unit_cost ) - $item->discount_amount + $item->tax_amount;
+			},
+			0
+		);
+		$level3_total += $result['shipping_amount'];
+
+		$this->assertEquals( 1560, $level3_total );
+	}
```

PHPMD (2.15.0)

[error] The method test_level3_data_with_negative_fee is not named in camelCase. (undefined)

(CamelCaseMethodName)